README.md

```diff
@@ -47,6 +47,69 @@ This application sets a custom user agent string of
 
 ## Deployment
 
-The application has a `manifest.yml` and can be deployed to the PaaS. You
-shouldn't need to do this yourself. Instead, raise a ticket with GDS Reliability
-Engineering to add the existing Service Broker to your Space.
+### Pre-requisites
+
+In order to deploy this on PaaS you will need to set up a [user provided service][] containing the following block of credentials - 
+
+```shell
+cf cups prometheus-targets-access -p '{
+  "AWS_ACCESS_KEY_ID":"<AWS ACCESS KEY ID>",
+  "AWS_SECRET_ACCESS_KEY":"<AWS SECRET>",
+  "UAA_PASSWORD":"<UAA PASSWORD>",
+  "UAA_USERNAME":"<UAA USERNAME>"
+}'
+```
+
+This block of credentials can be found in the team `reng-pass` credentials store.
+
+### Deploying the service broker
+
+The application has a `manifest.yml` which has cloud foundry config and can be deployed to the PaaS by running `cf push`, this should create a `prometheus-service-broker` and a `prometheus-target-updater` app.
+
+### Developing the service broker
+
+You will probably also want to set up a different [user provided service](#pre-requisites) so that your changes do not affect the live settings. Information to get the staging settings can be found in the `reng-pass` credentials store, if you change the name of the service from the default, make sure that the name is updated in the `manifest.yml`.
```

Comment:
I haven't really looked into setting up environment variables but it probably feels like a good thing to do as I was simply trying to improve the docs rather than make changes to the code. 

The bucket name, service id, plan id and service name would be stored in the `manifest.yml` file instead?
I'm not sure if the application name for the service broker and target updater can be have their values retrieved from env vars in the `manifest.yml`, we might have to use a `Makefile` in to generate the manifest with test settings for the user so will require more thought.

Regarding the `services` in the `manifest.yml`, that service name refers to the user provided service, needs to be the same as the the user provided service name created that holds the credentials.